addons/web/static/src/js/libs/zoomodoo.js

```diff
@@ -48,7 +48,10 @@ var defaults = {
     onHide: $.noop,
 
     // Callback function to execute when the cursor is moved while over the image.
-    onMove: $.noop
+    onMove: $.noop,
```

Comment:
it would have been nice to move the file to web/ and modify in two separated commits (to make it easier to track changes)